src/lib/components/content/SharedDetail.svelte

```diff
@@ -47,6 +49,29 @@
     <strong class="name">{name}</strong>
   {/if}
 
+  {#if removed}
+    <div class="removal-warning">
+      <div class="icon">
+        <WarningIcon />
+      </div>
+
+      <div class="warning">
+        <p><em>Possibly Unavailable</em></p>
+        <p>
+          This item or power has been marked as unavailable. This might be due to the item being
+          disabled temporarily, or it might be because the item has been permanently retired.
+        </p>
+        <p>
+          Think this warning should be removed? Let us know on our <a
+            href="https://discord.gg/JNCx6U3g9F"
+            target="_blank"
+            rel="noopener noreferrer">Discord</a
+          >.
+        </p>
+      </div>
+    </div>
+  {/if}
```

Comment:
I would personally extract this to a separate component, just to avoid the extra bulk